lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +91,50 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
+            config = json.load(file)
+        return ConfigDTO(
+            seed_urls=config["seed_urls"],
+            total_articles_to_find_and_parse=config["total_articles_to_find_and_parse"],
+            headers=config["headers"],
+            encoding=config["encoding"],
+            timeout=config["timeout"],
+            should_verify_certificate=config["should_verify_certificate"],
+            headless_mode=config["headless_mode"]
+        )
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
+            conf = json.load(file)
+
+        if not (isinstance(conf['seed_urls'], list)
+                and all(re.match(r"https?://(www.)?", seed_url)
```

Comment:
too wide check. maybe I put nn.ru in the config - do you really agree to parse it?

Reply:
The documentation asks to check this standart pattern, so I don't understand what I should avoid

Reply:
you should make your program behave accordingly. 

Scanarios:

1. user provides url to target website - OK
2. user provides url to website you do not support - they get appropriate error (and not like NoneType does not have property X).  Fail fast

Reply:
Ok, rewrote it